ui-particles-angular/projects/ui-particles-angular/src/lib/gio-submenu/gio-submenu.component.ts

```diff
@@ -13,11 +13,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component } from '@angular/core';
+import { Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+
+import { GioMenuService } from '../gio-menu/gio-menu.service';
 
 @Component({
   selector: 'gio-submenu',
   templateUrl: './gio-submenu.component.html',
   styleUrls: ['./gio-submenu.component.scss'],
 })
-export class GioSubmenuComponent {}
+export class GioSubmenuComponent implements OnInit {
+  public reduced = false;
+  public overlay = false;
+  private hover = false;
+  @ViewChild('gioSubmenu', { static: false })
+  private gioSubmenu: ElementRef<HTMLDivElement> | undefined;
+
+  constructor(private readonly gioMenuService: GioMenuService) {}
+
+  public ngOnInit(): void {
+    this.gioMenuService.reduce.subscribe(reduced => {
+      this.reduced = reduced;
+    });
+    this.gioMenuService.mouseOver.subscribe(mouseOver => {
+      if (this.gioSubmenu) {
+        this.overlay = mouseOver.enter;
+        if (this.overlay && this.reduced) {
+          this.gioSubmenu.nativeElement.style.top = `${mouseOver.top}px`;
+          this.gioSubmenu.nativeElement.style.height = 'auto';
+          this.gioSubmenu.nativeElement.style.maxHeight = `calc(100vh - ${mouseOver.top + 8}px)`;
+        } else if (!this.hover) {
+          this.gioSubmenu.nativeElement.style.height = '100%';
+          this.gioSubmenu.nativeElement.style.maxHeight = 'auto%';
```

Comment:
On the temporary storybook (https://6183b02d73381a003a3be1a6-bidgvwxfpn.chromatic.com/?path=/story/components-menu--with-sub-menu) , the submenu goes to the bottom. 
![image](https://user-images.githubusercontent.com/13161768/200508564-c390809d-7d2d-46f4-a83c-865d96237931.png)

Could we check this part ?

Reply:
The good link https://6183b02d73381a003a3be1a6-svoxvsxznn.chromatic.com/?path=/story/components-menu--with-sub-menu